odxtools/table.py

```diff
@@ -139,13 +137,12 @@ def __init__(self,
                  odx_id: OdxLinkId,
                  short_name: str,
                  table_rows: List[TableRow],
-                 table_row_refs: Optional[List[OdxLinkRef]] = None,
-                 long_name: Optional[str] = None,
-                 key_dop_ref: Optional[OdxLinkRef] = None,
-                 description: Optional[str] = None,
-                 semantic: Optional[str] = None,
-                 sdgs: List[SpecialDataGroup] = [],
-                 ):
+                 table_row_refs: Optional[List[OdxLinkRef]],
```

Comment:
Since all fields are now mandatory, lists should be initialized with empty lists instead of None

```
table_row_refs: List[OdxLinkRef],
```

Reply:
oops, true...

Reply:
(will be fixed in the next PR)